src/Client.php

```diff
@@ -68,20 +76,42 @@ public function __construct(string $endpoint, array $options = [], HttpClient $h
             ->setAllowedTypes('unfurl_media', 'bool')
             ->setAllowedTypes('allow_markdown', 'bool')
             ->setAllowedTypes('markdown_in_attachments', 'array')
+            ->setAllowedTypes('oauth_token', ['string', 'null'])
```

Comment:
Should be mandatory. Maybe it can be set on the constructor instead to be an option.

Reply:
It's not mandatory so as to not break backward compatibility.